README.md

```diff
@@ -1,4 +1,8 @@
 # ILGPU
+![CI](https://github.com/m4rs-mt/ILGPU/actions/workflows/ci.yml/badge.svg?style=flat)
```

Comment:
Maybe we want to refine our badge to link to the workflow runs and be only for `push` events to `master`?

```suggestion
[![CI](https://github.com/m4rs-mt/ILGPU/actions/workflows/ci.yml/badge.svg?branch=master&event=push)](https://github.com/m4rs-mt/ILGPU/actions/workflows/ci.yml)
```

Reply:
> Maybe we want to refine our badge to link to the workflow runs and be only for `push` events to `master`?

sounds very good, wasn't aware of that possibility ⚡

Reply:
actually the link to the workflow runs should also be filtered for `push` events to `master` only: https://github.com/m4rs-mt/ILGPU/actions/workflows/ci.yml?query=branch%3Amaster+event%3Apush